CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Implement `idyntree-model-view-meshcat` application. (https://github.com/robotology/idyntree/pull/961)
+### Fixed
+- Once `VisualElement::childElementForName` is called with input equal to `material` a pointer to the `materialInfo` is stored inside the `VisualElement`. (https://github.com/robotology/idyntree/pull/961)
```

Comment:
```suggestion
- Fix loading material and color information from URDF files. (https://github.com/robotology/idyntree/pull/961)
```

Reply:
(small comment, what matter to the high level user is the high level effect of the changes, not the low-level details with which really few user may be familiar with)